### DISCUSSION TITLE
Heroicons React no Typescript support

### DISCUSSION THREAD

**Discussion by JamDev0** [tailwindlabs/heroicons#1020] [Help]
I found an issue where it was said that Heroicons now have type definitions by default. But when I import it in my Next project, TypeScript throws the following error: ```Could not find a declaration file for module ‘@heroicons/react/outline. c:/Users/junbi/Documents/Programming/JDG/rede-social-front-end/node_modules/@heroicons/react/outline/index.js implicitly has an ‘any’ type. Try npm i --save-dev @types/heroicons__react if it exists or add a new declaration (.d.ts) file containing declare module '@heroicons/react/outline```.
Also, the npm page dosen't shows that the package has support for typescript

**Comment by abcdefghijorngarbosaxyz** [comment]
It's `import { SomeIcon } from "@heroicons/react/24/outline"`.

**Comment by MirstT** [comment]
Add this line to a .d.ts file in your project:
`declare module '@heroicons/*';`
